lib/loaders/babel.js

```diff
@@ -55,14 +55,13 @@ module.exports = {
                 presets: [
                     [require.resolve('@babel/preset-env'), presetEnvOptions]
                 ],
-                plugins: [require.resolve('@babel/plugin-syntax-dynamic-import')]
+                plugins: []
```

Comment:
I keep it even if empty, in order to prevent more breaking changes, if people used `config.plugins.push()` like we did in tests.